### PR TITLE
feat: add eventbridge cloudevents transformer pattern

### PR DESCRIPTION
**Pull request by embano1**
*Issue #, if available:*

Closes: #1433

*Description of changes:*

Add new pattern describing how to use EventBridge input transformations to convert EventBridge events to CloudEvents.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eventbridge-cloudevents-transformer-cdk-typescript/src/test/eb-lambda-cloudevents.test.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as EbLambdaCloudevents from '../lib/eb-lambda-cloudevents-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/eb-lambda-cloudevents-stack.ts
+test('SQS Queue Created', () => {
```
Comment (boyney123) [inline eventbridge-cloudevents-transformer-cdk-typescript/src/test/eb-lambda-cloudevents.test.ts:7]: Think we can remove this file @embano1 ?

**Comment by boyney123** [comment]
Very cool @embano1 thanks! 1 comment and failed schema to check, rest is looking good mate.

File: eventbridge-cloudevents-transformer-cdk-typescript/src/test/eb-lambda-cloudevents.test.ts (hunk 1 above)
Comment (embano1) [reply eventbridge-cloudevents-transformer-cdk-typescript/src/test/eb-lambda-cloudevents.test.ts:7]: Yeah, everyone had it in so I though #YOLO :D

**Comment by embano1** [comment]
Thy @boyney123 for your quick review! Feedback addressed!

**Comment by boyney123** [comment]
Thanks @embano1 !

**Pull request merged**